### PR TITLE
Add base16 shell

### PR DESCRIPTION
**Pull request by ErickTucto**
Adding base16 shell on xontrib https://github.com/xonsh/xonsh/issues/1228#issuecomment-446711013

### CODE REVIEW

File: news/base16-shell.rst [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+
+    .. code:: python
+        :number-lines:
+
+        $BASE16_SHELL = $HOME + "/.config/base16-shell/"
+        xontrib load base16_shell
+
+See image
+
+|Terminal|
```
Comment (scopatz) [inline news/base16-shell.rst]: I believe that needs to be indented.

**Comment by scopatz** [comment]
Thanks @ErickTucto! The tests are failing because the news file isn't valid rst

**Comment by ErickTucto** [comment]
Thanks @scopatz . I will correct it.

File: news/base16-shell.rst (hunk 1 above)
Comment (ErickTucto) [reply news/base16-shell.rst]: https://github.com/xonsh/xonsh/blob/51d9f77fd2aee6706cefd079d4fd93be7ebeff0f/news/base16-shell.rst#L3

I wrote the link incorrectly

**Comment by scopatz** [comment]
Awesome thanks!

**Pull request merged**